pylint/checkers/refactoring/recommendation_checker.py

```diff
@@ -60,6 +60,13 @@ class RecommendationChecker(checkers.BaseChecker):
             "which could potentially be a f-string. The use of f-strings is preferred. "
             "Requires Python 3.6 and ``py-version >= 3.6``.",
         ),
+        "C0210": (
+            "Consider using 'while %s' instead of 'while True:' an 'if', and a 'break'",
+            "consider-refactoring-into-while-condition",
+            "Emitted when `while True:` loop is used and the first statement is a break condition. "
+            "The ``if / break`` construct can be removed if the check is inverted and moved to "
+            "the ``while`` statement.",
+        ),
```

Comment:
To make this an extension this part must be moved in [pylint/extensions](https://github.com/PyCQA/pylint/tree/main/pylint/extensions) instead of ``pylint/checkers`` along with the ``visit_while`` function.